tensorflow/core/kernels/linalg/matrix_solve_op.cc

```diff
@@ -290,19 +300,27 @@ class MatrixSolveOpGpu : public AsyncOpKernel {
     auto transposed_rhs_reshaped =
         transposed_rhs.template flat_inner_dims<Scalar, 3>();
     if (use_batched_solver) {
+#if GOOGLE_CUDA
       const Scalar** input_copy_ptrs_base =
           reinterpret_cast<const Scalar**>(input_copy_ptr_array.mutable_data());
       const Scalar** transposed_rhs_ptrs_base =
           reinterpret_cast<const Scalar**>(
               transposed_rhs_ptr_array.mutable_data());
+#else //TENSORFLOW_USE_ROCM
```

Comment:
what's the difference between CUDA/ROCm here? Just `const`? I would drop it for both then.

Reply:
Resolved.